Backend/lib/shared-services/src/IssueManagement/Tools/UtilsTH.hs

```diff
@@ -0,0 +1,33 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module IssueManagement.Tools.UtilsTH (module Reexport, module IssueManagement.Tools.UtilsTH) where
+
+import Kernel.Beam.Lib.UtilsTH as Reexport hiding (mkTableInstances, mkTableInstancesWithTModifier, schemaName)
+import qualified Kernel.Beam.Lib.UtilsTH as TH
+import Kernel.Prelude hiding (catch)
+import Kernel.Types.Common as Reexport hiding (id)
+import Language.Haskell.TH
+import System.Environment (getEnv)
+import System.IO.Unsafe (unsafePerformIO)
+
+schemaName :: String
+{-# NOINLINE schemaName #-}
+schemaName = unsafePerformIO (getEnv "SCHEMA_NAME")
```

Comment:
Is it acceptable for us to fetch `SCHEMA_NAME` from env variables? @hkmangla 
What if we use `run-mobility-stack-dev ` command locally? Then the same `SCHEMA_NAME` will be used both for bap and bpp, right?

Reply:
Remove this way of fetching schemaName.